RemoteWellnessSupportApp/Condition/ViewModifiers/PhysicalConditionChartYModifier.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  PhysicalConditionChartYModifier.swift
+//  RemoteWellnessSupportApp
+//
+//  Created by 岩本雄貴 on 2024/02/27.
+//
+
+import Charts
+import SwiftUI
+
+struct PhysicalConditionChartYModifier: ViewModifier {
+    let physicalConditionRateRange = 0 ... 5
+    let ratingValues: [Int]
+
+    func body(content: Content) -> some View {
+        content
+            .chartYScale(domain: physicalConditionRateRange)
+            .chartYAxis {
+                AxisMarks(position: .leading, values: ratingValues) { value in
+                    if let rawValue = value.as(Int.self) {
```

Comment:
ややifのネストが深いですが、ひとまとめにすることができます。

```suggestion
                    if let rawValue = value.as(Int.self), let rating = PhysicalConditionRating(rawValue: rawValue) {
```

Reply:
こちらで対応しました
https://github.com/y-iwamoto/RemoteWellnessSupportApp/pull/7/commits/5c65b819fbdc2fac2b1392ce2d71952e1e092306